config/ember-try.js

```diff
@@ -2,8 +2,36 @@
 
 const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 
+const safe = embroiderSafe();
+const optimized = embroiderOptimized();
+
 module.exports = async function () {
   return {
-    scenarios: [embroiderSafe(), embroiderOptimized()],
+    scenarios: [
+      {
+        ...safe,
+        npm: {
+          ...safe.npm,
+          devDependencies: {
+            ...safe.npm.devDependencies,
+            '@embroider/core': `3.0.2`,
+            '@embroider/webpack': `3.0.0`,
+            '@embroider/compat': `3.0.2`,
+          },
+        },
+      },
+      {
+        ...optimized,
+        npm: {
+          ...optimized.npm,
+          devDependencies: {
+            ...optimized.npm.devDependencies,
+            '@embroider/core': `3.0.2`,
+            '@embroider/webpack': `3.0.0`,
+            '@embroider/compat': `3.0.2`,
+          },
+        },
+      },
+    ],
```

Comment:
On pourrait variabiliser les dependences pour eviter toute confusion

```suggestion
 const pinnedDependencies = [
            '@embroider/core': `3.0.2`,
            '@embroider/webpack': `3.0.0`,
            '@embroider/compat': `3.0.2`,
    ];
    scenarios: [
      {
        ...safe,
        npm: {
          ...safe.npm,
          devDependencies: {
            ...safe.npm.devDependencies,
           ...pinnedDependencies,
          },
        },
      },
      {
        ...optimized,
        npm: {
          ...optimized.npm,
          devDependencies: {
            ...optimized.npm.devDependencies,
            ...pinnedDependencies
          },
        },
      },
    ],
```